packages/model-typings/src/models/IModerationReportsModel.ts

```diff
@@ -17,7 +17,13 @@ export interface IModerationReportsModel extends IBaseModel<IModerationReport> {
 		reportedBy: IModerationReport['reportedBy'],
 	): ReturnType<IBaseModel<IModerationReport>['insertOne']>;
 
-	findReportsGroupedByUser(
+	createWithDescriptionAndUser(
+		reportedUser: Exclude<IModerationReport['reportedUser'], undefined>,
+		description: string,
+		reportedBy: IModerationReport['reportedBy'],
+	): ReturnType<IBaseModel<IModerationReport>['insertOne']>;
+
+	findMessageReportsGroupedByUser(
```

Comment:
We should also ensure that this will only return message reports (with `message: { $exists: true }` or something similar)

Reply:
Fixed. I added this check to the `getSearchQueryForSelector` method.